native/game_backend/src/map.rs

```diff
@@ -96,14 +104,16 @@ pub fn collision_with_edge(center: &Position, size: u64, width: u64, height: u64
     false
 }
 
-pub fn random_position(width: u64, height: u64) -> Position {
+pub fn random_position(width: u64, _height: u64) -> Position {
     let rng = &mut rand::thread_rng();
-    let bound_x = (width / 2) as i64;
-    let bound_y = (height / 2) as i64;
+
+    let radius = (width / 2) as i64;
+    let random_radius = (rng.gen_range(0..radius.pow(2)) as f32).sqrt();
```

Comment:
why do we need the pow(2) and sqrt? can't. we just use `rng.gen_range(0..radius)`?

Reply:
Because if not, we get a nonuniform distribution:
https://mathworld.wolfram.com/DiskPointPicking.html